src/main/java/core/basesyntax/Splitter.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Splitter {
+    public ReportItem splitString(String string) {
+        String[] stringArray = string.split(",");
+        Name name = Name.BUY.name().equalsIgnoreCase(stringArray[0]) ? Name.BUY : Name.SUPPLY;
```

Comment:
The use of `Name.BUY.name().equalsIgnoreCase(stringArray[0])` is a bit complex and could be simplified for readability. Also, consider handling the case where 'stringArray[0]' does not match any of the expected 'Name' enum values.